src/styles/block/nav.scss

```
@@ -0,0 +1,36 @@
+.nav {
+  &__list {
+    margin: 0;
+    padding: 0;
+    list-style: none;
+    display: flex;
+    flex-direction: column;
+    gap: 32px;
+  }
+
+  &__link {
+    color: #{$color-text};
```

try to use the same format for all variables
```suggestion
    color: $color-text;
```